src/main/java/spim/io/OMETIFFHandler.java

```diff
@@ -54,7 +54,10 @@ public OMETIFFHandler(CMMCore iCore, File outDir, String xyDev,
 		deltat = iDeltaT;
 		outputDirectory = outDir;
 		this.acqRows = acqRows;
-
+		tiles = tileCount;
+		int angles = Math.floorDiv(stacks, tiles);
```

Comment:
@pawlowska Could you update the codes as @dscho explained above? Then, I will merge it to master. Thank you!